tools/GrammarTesting/Tests/Parsing/Samples/v8/Arrays of nullables/sample.cs

```diff
@@ -0,0 +1,36 @@
+class ArraysOfNullable
+{
+   class A { }
+
+   void ArraysOfNullable()
+   {
+      A[] p_1 = new A[5];    // Array of non-nullable A all elements inited to null (oops)
+      A[]? q_1 = null;       // Nullable array of non-nullable A, array inited to null
+      A?[] r_1 = new A[5];   // Array of nullable A all elements inited to null
+      A?[]? s_1 = null;      // Nullable array of nullable A, array inited to null
```

Comment:
@Nigel-Ecma Would you be able to show me how to work with files like this? This is from @gafter's PR and I copied it across, and I'd like to add some new cases when I work with it experimentally.

I don't yet understand what all can be done with this. The other folders appear to have had a tool run on them which generates a graphical representation of the parse. I'd like to know how to run it. Does a PR fail if the generated parse is out of date with the sample.cs file?